oauth2_http/java/com/google/auth/oauth2/ExternalAccountCredentials.java

```diff
@@ -808,11 +795,12 @@ public Builder setTokenUrl(String tokenUrl) {
     /**
      * Sets the external credential source.
      *
-     * @param credentialSource the {@code CredentialSource} to set
+     * @param externalAccountCredentialSource the {@code CredentialSource} to set
      * @return this {@code Builder} object
      */
-    public Builder setCredentialSource(CredentialSource credentialSource) {
-      this.credentialSource = credentialSource;
+    public Builder setCredentialSource(
+        ExternalAccountCredentialSource externalAccountCredentialSource) {
```

Comment:
this is breaking right?

Reply:
should we keep an old CredentialSource temporarily?

Reply:
keeping CredentialSource for now